### PR TITLE
mgmt, local generation for network 2023-11-01

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to release [issue#5220](https://github.com/Azure/sdk-release-request/issues/5220)

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json [hunk 1]
```diff
@@ -484,11 +484,22 @@
           "justification": "Removing Jackson annotations from Tables in transition to stream-style."
         },
         {
-          "regex": true,
```
Comment (weidongxu-microsoft) [inline eng/code-quality-reports/src/main/resources/revapi/revapi.json:487]: why modify this?

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json [hunk 2]
```diff
@@ -484,11 +484,22 @@
...
           "code": "java\\.annotation\\.removed",
           "old": ".*? com\\.azure\\.communication\\.jobrouter\\.models.*",
           "new": ".*? com\\.azure\\.communication\\.jobrouter\\.models.*",
           "justification": "Removing Jackson annotations from Job Router in transition to stream-style."
+        },
+        {
+          "regex": true,
+          "code": "java\\.annotation\\.attributeRemoved",
+          "old": ".*? com\\.azure\\.resourcemanager\\.network\\.models.*",
+          "justification": "Removing Jackson annotations from Network in transition to stream-style."
```
Comment (weidongxu-microsoft) [inline eng/code-quality-reports/src/main/resources/revapi/revapi.json:496]: wrong reason.

Just say "Jackson annotation changed" should be enough.

**Review by weidongxu-microsoft — APPROVED** [review]

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json (hunk 2 above)
Comment (v-hongli1) [reply eng/code-quality-reports/src/main/resources/revapi/revapi.json]: Fixed in the new version.

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json (hunk 1 above)
Comment (v-hongli1) [reply eng/code-quality-reports/src/main/resources/revapi/revapi.json:487]: Merge `revapi` configuration for `attributeRemoved` and `attributeAdded`.

**Review by weidongxu-microsoft — APPROVED** [review]

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**